scripts/add-install-comment.js

```diff
@@ -10,7 +10,8 @@ const {
   getCIName,
```

Comment:
The changes in this file bypass/fix https://github.com/cypress-io/release-automations/issues/67, which allows us to remove a package from release-automations: https://github.com/cypress-io/release-automations/pull/69